### DISCUSSION TITLE
Invalid name '', must include alphanumeric characters errors with MDX format

### DISCUSSION THREAD

**Discussion by coreybruyere** [storybookjs/storybook#12938] [Help]
I have a few different MDX files that work perfectly fine. I'm creating a new Heading component that I'm testing and am getting these errors:
`Uncaught Error: Invalid name '', must include alphanumeric characters`
&
`The above error occurred in the <Canvas> component`

Anyone see anything that looks off in this WIP PR I have: https://github.com/coreybruyere/skeleton-ui/pull/6/files

**Comment by coreybruyere** [comment]
Closing the Meta component with another tag seemed to fix the issue in the MDX files that weren't rendering and displaying that error. Kind of weird that no other MDX storybook files had that issue. 

```
<Meta title="Components/Heading" component={Heading}></Meta>
```

**Reply by jingoldby** [reply]
T h a n k y o u

**Reply by MickL** [reply]
Can this be part of the documentation? https://storybook.js.org/docs/angular/api/mdx

Didnt fix the issue for me tho

**Reply by shilman** [reply]
@coreybruyere Before you made the change did you have a self-closing Meta like `<Meta ... />`? If you didn't include the trailing slash that would def cause problems.

**Reply by coreybruyere** [reply]
Yea I had a trailing slash.
